app/Main.hs

```diff
@@ -1,7 +1,176 @@
-module Main (main) where
+{-# OPTIONS_GHC -Wno-incomplete-record-updates #-}
```

Comment:
Can you move `ClientInput` to a different file so that we keep this as contained as possible?